app/views/content/funding-and-support/salaried-teacher-training.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Salaried teacher training"
+description: |-
+    Explore salaried teacher training, including School Direct (salaried) courses, postgraduate teaching apprenticeships and training delivered by Teach First.
+related_content:
+    Nathan's experience of salaried teacher training: "/blog/salaried-teacher-training-classroom-learning"
+    If you've worked as an unqualified teacher: "/train-to-be-a-teacher/assessment-only-route-to-qts"
+    What to expect on your teacher training: "/blog/what-to-expect-on-your-teacher-training"
+promo_content:
+    - content/train-to-be-a-teacher/promos/find-your-course
+    - content/funding-and-support/promos/get-adviser-funding-promo
+navigation: 20.20
+navigation_description: Find out about salaried teacher training, including School Direct (salaried) courses, postgraduate teaching apprenticeships and training delivered by Teach First.
```

Comment:
Do we need to say 'salaried teacher training' as that's in the heading? Maybe something like:
```suggestion
navigation_description: Find out about School Direct (salaried) courses, postgraduate teaching apprenticeships and training delivered by Teach First.
```